### PR TITLE
Drpai translator

### PR DESCRIPTION
**Pull request by mistyron**
Added drpai-translator container imageand added `paths:` variable to all the worlkflow files to only run for contents in `"build/**" `

### CODE REVIEW

**Comment by mistyron** [comment]
> Each time these files are changed, the RZV2L images will be built which is unrelated to the contents of these files. So I think this folder should become a separate repository and it should have a different CI/CD.

I see what you are saying and don't diagree! The initial idea was tohave various directories & containers in this repo (hence the `Build/`). Things have developed since the initial idea :)

**Comment by matinlotfali** [comment]
> The initial idea was to have various directories & containers in this repo

So I suggest we limit the current GitHub Actions to the `build/` directory. This means if files outside of the directory has changed, it won't run it.

**Comment by mistyron** [comment]
> > The initial idea was to have various directories & containers in this repo
> 
> So I suggest we limit the current GitHub Actions to the `build/` directory. This means if files outside of the directory has changed, it won't run it.

Yes, I agree with that!

**Comment by matinlotfali** [comment]
Your finding is correct and I agree.

**Comment by mistyron** [comment]
> Your finding is correct and I agree.

Okay, I have gone ahead and updated the `yml` files accordingly. I'm not exactly sure how to verify and test the changes though.

**Comment by matinlotfali** [comment]
I think they should be capital B.
Linux directories are case-sensitive.

**Comment by mistyron** [comment]
> I think they should be capital B. Linux directories are case-sensitive.

DUH, of course! Good catch! I'll fix!

**Comment by mistyron** [comment]
> I think they should be capital B. Linux directories are case-sensitive.

Give it another look when you can!

File: .github/workflows/release-submission.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ on:
         branches: [master]
         types:
             - completed
+        paths: [ "Build/**" ]
```
Comment (matinlotfali) [inline .github/workflows/release-submission.yml:9]: I think this one can be non-specific for directories

File: .github/workflows/master-merge.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ name: Merge master if all reviews are approved
 on:
   pull_request_review:
     types: [submitted]
+    paths: [ "Build/**" ]
```
Comment (matinlotfali) [inline .github/workflows/master-merge.yml:6]: I think this one can be non-specific for directories

File: .github/workflows/develop-ff.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ name: FastForward Develop to match Master
 on:
   push:
     branches: [ "master" ]
+    paths: [ "Build/**" ]
```
Comment (matinlotfali) [inline .github/workflows/develop-ff.yml:6]: I think this one can be non-specific for directories

File: .github/workflows/create-pull-request.yml [hunk 4]
```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - develop
+    paths: [ "Build/**" ]
```
Comment (matinlotfali) [inline .github/workflows/create-pull-request.yml:6]: I think this one can be non-specific for directories

**Review by matinlotfali — CHANGES_REQUESTED** [review]
I think some of the GitHub Actions are suitable for the whole repository. Only two of them are specific to the Build directory.

**Comment by mistyron** [comment]
> I think some of the GitHub Actions are suitable for the whole repository. Only two of them are specific to the Build directory.

Oh, I assumed all the workflows would be for stuff in `Build/` only - but I've now changed the `yml`s accordingly

**Review by matinlotfali — APPROVED** [review]
It looks good now

**Pull request merged**